Source/FolioReaderFontsMenu.swift

```diff
@@ -234,7 +234,14 @@ class FolioReaderFontsMenu: UIViewController, SMSegmentViewDelegate, UIGestureRe
         layoutDirection.tag = 3
         layoutDirection.addSegmentWithTitle(readerConfig.localizedLayoutVertical, onSelectionImage: verticalSelected, offSelectionImage: verticalNormal)
         layoutDirection.addSegmentWithTitle(readerConfig.localizedLayoutHorizontal, onSelectionImage: horizontalSelected, offSelectionImage: horizontalNormal)
-        layoutDirection.selectSegmentAtIndex(Int(FolioReader.currentScrollDirection))
+
+        var scrollDirection = FolioReaderScrollDirection(rawValue: FolioReader.currentScrollDirection)
+
+        if scrollDirection == .defaultVertical && readerConfig.scrollDirection != .defaultVertical {
```

Comment:
Same as [this comment](https://github.com/FolioReader/FolioReaderKit/pull/178/files#r91723848).